src/App.tsx

```diff
@@ -15,36 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  LENGTH = 'L',
+  ALPHABET = 'A',
+  DEFAULT = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  isSorted: string,
+  isReversed: boolean,
+): string[] {
+  const preperedGoods = [...goods];
```

Comment:
The variable `preperedGoods` is misspelled. It should be `preparedGoods`.

Reply:
> The variable `preperedGoods` is misspelled. It should be `preparedGoods`.

Good point